workbench-core/repo-scripts/github-oidc/README.md

```diff
@@ -0,0 +1,31 @@
+# `github-oidc`
+
+⚠️ $\textcolor{red}{\text{Experimental}}$ ⚠️ : Not for use in any critical, production, or otherwise important deployments
+
+## Description
+This package creates the roles in an AWS account that can be assumed from a github workflow. The permissions given to these roles enable the workflow to perform deployments in the specified AWS account. 2 CDK stacks are created as part of this package:
+1. `OIDCProviderStack`: Creates the Identity Provider
```

Comment:
Please expand on what this deploys -- looks like it deploys a lambda? (according to nag below)

Reply:
The lambda suppression that you see as part of the nag is the CustomResource created by `OpenIdConnectProvider` construct (The Lambda can be treated as CDK internal).

This stack just creates the Identity Provider for Github